.github/workflows/test.yml

```diff
@@ -0,0 +1,15 @@
+name: CI
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Tests
+    uses: adamruzicka/actions/.github/workflows/test-gem.yml@test-gem-packages
```

Comment:
We introduced https://github.com/theforeman/actions/blob/v0/.github/workflows/smart_proxy_plugin.yml but didn't add this to our README.